### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by dvirk**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -8,4 +8,5 @@ PyQt5==5.10.1
 pytest>=3.7
 pytest-cov>=2.5.1
 pytest-qt==3.0.0
+pytest-xvfb >=1.1.0
```
Comment (mick-warehime) [inline requirements.txt]: what does this do?

File: src/controller/decision_controller.py [hunk 2]
```diff
@@ -19,6 +19,7 @@ def __init__(self, data: DecisionSceneData, *args: Any,
                  **kwargs: Any) -> None:
         super().__init__(*args, **kwargs)
 
+        self.buttons = []  # type: List[QPushButton]
```
Comment (mick-warehime) [inline src/controller/decision_controller.py]: my prefererence

self.buttons: List[QPushButton] = []

File: src/tests/test_decision_controller.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+
+def test_decision_controller_choice_resolution(qtbot):
+    main = MainWindow()
+
+    main.signals.load_scene.emit('start')
+
+    controller = main.takeCentralWidget()
+
+    assert isinstance(controller, DecisionController)
```
Comment (mick-warehime) [inline src/tests/test_decision_controller.py]: put this below - to keep the approach

setup 

execute

assert

File: requirements.txt (hunk 1 above)
Comment (dvirk) [reply requirements.txt]: It prevents travis from popping up the gui during tests.

File: src/controller/decision_controller.py (hunk 2 above)
Comment (dvirk) [reply src/controller/decision_controller.py]: i forgot you could do that.

File: src/tests/test_decision_controller.py (hunk 3 above)
Comment (dvirk) [reply src/tests/test_decision_controller.py]: If this were to fail, I would want it to fail before line 22 is reached.

**Pull request merged**